tests/test_user_management.py

```diff
@@ -265,21 +292,54 @@ def test_update_user_password(self, mock_user, capture_and_mock_request):
         assert user["id"] == "user_01H7ZGXFP5C6BBQY6Z7277ZCT0"
         assert request["json"]["password"] == "pass_123"
 
-    def test_add_user_to_organization(self, capture_and_mock_request, mock_user):
-        url, _ = capture_and_mock_request("post", mock_user, 200)
+    def test_create_organization_membership(
+        self, capture_and_mock_request, mock_organization_membership
+    ):
+        user_id = "user_12345"
+        organization_id = "org_67890"
+        url, _ = capture_and_mock_request("post", mock_organization_membership, 201)
 
-        user = self.user_management.add_user_to_organization("user_123", "org_123")
+        organization_membership = self.user_management.create_organization_membership(
+            user_id=user_id, organization_id=organization_id
+        )
 
-        assert url[0].endswith("users/user_123/organization/org_123")
-        assert user["id"] == "user_01H7ZGXFP5C6BBQY6Z7277ZCT0"
+        assert url[0].endswith("user_management/organization_memberships")
+        assert organization_membership["user_id"] == user_id
+        assert organization_membership["organization_id"] == organization_id
+
+    def test_get_organization_membership(
+        self, mock_organization_membership, capture_and_mock_request
+    ):
+        url, request_kwargs = capture_and_mock_request(
+            "get", mock_organization_membership, 200
+        )
 
-    def test_remove_user_from_organization(self, capture_and_mock_request, mock_user):
-        url, _ = capture_and_mock_request("delete", mock_user, 200)
+        om = self.user_management.get_organization_membership("om_ABCDE")
 
-        user = self.user_management.remove_user_from_organization("user_123", "org_123")
+        assert url[0].endswith("user_management/organization_memberships/om_ABCDE")
+        assert om["id"] == "om_ABCDE"
 
-        assert url[0].endswith("users/user_123/organization/org_123")
-        assert user["id"] == "user_01H7ZGXFP5C6BBQY6Z7277ZCT0"
+    def test_list_organization_memberships_returns_metadata(
```

Comment:
what's the purpose of this test?

Reply:
I copied this from `list_users` - my understanding is that it's testing that the list method is parsing the data and adding the metadata. But yeah let me look at our tests for list methods and see what might be appropriate here.

Reply:
yeah ideally we'd have a test for the list method that asserts on the user-facing parts